fhir-server/src/main/java/com/ibm/fhir/server/filter/rest/FHIRHttpServletRequestWrapper.java

```diff
@@ -763,247 +487,19 @@ public String getQueryString() {
-     */
-    @Override
-    public HttpSession getSession() {
-        return delegate.getSession();
-    }
-
-    /**
-     * @param arg0
-     * @return
-     * @see javax.servlet.http.HttpServletRequest#getSession(boolean)
-     */
-    @Override
-    public HttpSession getSession(boolean arg0) {
-        return delegate.getSession(arg0);
-    }
-
-    /**
-     * @return
-     * @see javax.servlet.http.HttpServletRequest#getUserPrincipal()
-     */
-    @Override
-    public Principal getUserPrincipal() {
-        return delegate.getUserPrincipal();
-    }
-
-    /**
-     * @return
-     * @see javax.servlet.http.HttpServletRequest#isRequestedSessionIdFromCookie()
-     */
-    @Override
-    public boolean isRequestedSessionIdFromCookie() {
-        return delegate.isRequestedSessionIdFromCookie();
-    }
-
-    /**
-     * @return
-     * @see javax.servlet.http.HttpServletRequest#isRequestedSessionIdFromURL()
-     */
-    @Override
-    public boolean isRequestedSessionIdFromURL() {
-        return delegate.isRequestedSessionIdFromURL();
-    }
-
-    /**
-     * @return
-     * @deprecated
-     * @see javax.servlet.http.HttpServletRequest#isRequestedSessionIdFromUrl()
-     */
-    @Deprecated
-    @Override
-    public boolean isRequestedSessionIdFromUrl() {
-        return delegate.isRequestedSessionIdFromUrl();
-    }
-
-    /**
-     * @return
-     * @see javax.servlet.http.HttpServletRequest#isRequestedSessionIdValid()
-     */
-    @Override
-    public boolean isRequestedSessionIdValid() {
-        return delegate.isRequestedSessionIdValid();
-    }
-
-    /**
-     * @return
-     * @see javax.servlet.ServletRequest#isSecure()
-     */
-    @Override
-    public boolean isSecure() {
-        return delegate.isSecure();
-    }
-
-    /**
-     * @param arg0
-     * @return
-     * @see javax.servlet.http.HttpServletRequest#isUserInRole(java.lang.String)
-     */
-    @Override
-    public boolean isUserInRole(String arg0) {
-        return delegate.isUserInRole(arg0);
-    }
-
-    /**
-     * @param arg0
-     * @see javax.servlet.ServletRequest#removeAttribute(java.lang.String)
-     */
-    @Override
-    public void removeAttribute(String arg0) {
-        delegate.removeAttribute(arg0);
-    }
-
-    /**
-     * @param arg0
-     * @param arg1
-     * @see javax.servlet.ServletRequest#setAttribute(java.lang.String, java.lang.Object)
-     */
-    @Override
-    public void setAttribute(String arg0, Object arg1) {
-        delegate.setAttribute(arg0, arg1);
-    }
-
-    /**
-     * @param arg0
-     * @throws UnsupportedEncodingException
-     * @see javax.servlet.ServletRequest#setCharacterEncoding(java.lang.String)
-     */
-    @Override
-    public void setCharacterEncoding(String arg0) throws UnsupportedEncodingException {
-        delegate.setCharacterEncoding(arg0);
+    @SuppressWarnings("unused")
+    private String displayHeaderValues(Enumeration<String> headers) {
```

Comment:
this was there before, i just added the javadoc for it.  i think we could probably axe this one as well.